samples/cpp/benchmark_app/main.cpp

```diff
@@ -373,20 +374,19 @@ int main(int argc, char* argv[]) {
         // Update config per device according to command line parameters
         for (auto& device : devices) {
             auto& device_config = config[device];
-
-            // high-level performance modes
-            if (!device_config.count(ov::hint::performance_mode.name())) {
-                device_config.emplace(ov::hint::performance_mode(get_performance_hint(device, core)));
+            auto ov_perf_hint = get_performance_hint(device, core);
```

Comment:
If performance hint is set per device in config, it will be overridden here

Reply:
The config setting from cmd line should be higher priority than the config setting from JSON file, right? 
Overriding should be the expected behavior here if performance hint is already existed in the config.

Reply:
https://github.com/openvinotoolkit/openvino/blob/82a992b95d2264b8c26c5d3af6feb1d877bcf3ac/samples/cpp/benchmark_app/main.cpp#L142

Reply:
You mean we need to set hint to `throughput ` as default value into the config here if no command line setting for `performance_hint`?

Reply:
> You mean we need to set hint to `throughput ` as default value into the config here if no command line setting for `performance_hint`?

I mean we don't need to set hint to default value if it is present in config.

Reply:
Updated.